pkg/apis/unikorn/v1alpha1/types.go

```diff
@@ -692,7 +692,7 @@ type ServerStatus struct {
 	// Current service state of a cluster manager.
 	Conditions []unikornv1core.Condition `json:"conditions,omitempty"`
 	// Phase is the current lifecycle phase of the server.
-	Phase InstanceLifecyclePhase `json:"phase,omitempty"`
+	Phase *InstanceLifecyclePhase `json:"phase,omitempty"`
```

Comment:
Just FYI, as this is really a string, the zero value of "" will trigger nothing to be generated.  But mistakes can happen I guess, so the difference between set but empty, and not set may be handy.  Although it causes the code to be a bit more messy :/